src/Compilers/CSharp/Test/Semantic/Semantics/OverloadResolutionPerfTests.cs

```diff
@@ -350,7 +350,7 @@ static void F(IEnumerable<int[]> x)
             comp.VerifyDiagnostics();
         }
 
-        [Fact]
+        [ConditionalFact(typeof(NoIOperationValidation), Reason = "Timeouts")]
```

Comment:
AlekseyTs said:

> Should we have an issue to track disable tests like that?

Reply:
I don't think that's particularly useful, as I'm not currently thinking that we'll be attempting to renable them in the future.

---
In reply to: [571246466](https://github.com/dotnet/roslyn/pull/51032#discussion_r571246466) [](ancestors = 571246466)